components/sup/src/error.rs

```diff
@@ -40,6 +40,7 @@ pub enum Error {
     CtlSecretIo(PathBuf, io::Error),
     APIClient(habitat_api_client::Error),
     EnvJoinPathsError(env::JoinPathsError),
+    EnvVarError(env::VarError),
```

Comment:
It does not look like this is used anywhere.

Reply:
Well I needed the `From` trait implementation because of `?` at https://github.com/habitat-sh/habitat/pull/7840/files#diff-b7cc1dcc0589de0582a1250393299db1R195

Reply:
May bad :sweat_smile:; I was testing on linux.